modules/ztp-verifying-ipsec.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-advanced-install-ztp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-verifying-ipsec_{context}"]
+= Verifying the IPsec encryption
+
+You can verify that the IPsec encryption is successfully applied in a managed OpenShift cluster.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+* You have configured the IPsec encryption.
+
+.Procedure
+
+. Start a debug pod for the managed cluster by running the following command:
++
+[source,terminal]
+----
+$ oc debug node/<node_name>
+----
+
+. Check that the IPsec policy is applied in the cluster node:
```

Comment:
I think that even if these commands are being run from a shell as root, the [docs guidelines rule](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command) about explicitly mentioning commands still applies:
```suggestion
. Check that the IPsec policy is applied in the cluster node by running the following command:
```

Reply:
Updated as per the suggestion.